lib/actions.js

```diff
@@ -59,6 +65,9 @@ module.exports = {
 
   isEmailSendingAction: function isEmailSendingAction(action) {
     return EMAIL_SENDING_ACTION[action]
-  }
+  },
 
+  isSMSSendingAction: function isSMSSendingAction(action) {
```

Comment:
Removed the extra function names but couldn't go full ES6, https://github.com/mozilla/fxa-customs-server/issues/164.